### PR TITLE
Start dev guidelines

### PR DESCRIPTION
**Pull request by lubkoll**
I added placeholders for
* DoD
* ready for review

We can add these over the next weeks after discussing them.

### CODE REVIEW

File: DEV_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Development Guidelines
+
+## Pull requests
+In order to allow fast and efficient reviews, please create small PRs per independent change instead of big ones for a full feature. 
+
+**Rule of thumb**: It should be easy to formulate the commit message as a one-liner. If that is not the case, then the PR is probably to big.
+
+
+## DoD -- Definition of Done
+
```
Comment (arhamchordia) [inline DEV_GUIDELINES.md:10]: DOD for chain : 
1. Code Quality and Standards matches reviewers expectation.
2. Functionality justifies the title of the PR
3. Testing should include unit tests and end to end tests if applicable.
4. Documentation (this I feel depends on timeline of the issue being addressed)
5. CI steps should pass with all green checks

**Review by arhamchordia — CHANGES_REQUESTED** [review]

File: DEV_GUIDELINES.md (hunk 1 above)
Comment (arhamchordia) [reply DEV_GUIDELINES.md:10]: Also ready for review should mean that the above checks have been done by the person creating the PR.

Comment (lubkoll) [reply DEV_GUIDELINES.md:10]: As mentioned above these are to be discussed separately.

Comment (ajansari95) [reply DEV_GUIDELINES.md:10]: apart from above i think as we discussed basic guideline should be a adding one liner  explaining the "what?why?how?" for the corresponding change

**Review by magiodev — APPROVED** [review]

**Review by arhamchordia — APPROVED** [review]

**Comment by arhamchordia** [comment]
Why are all the commits force pushes?

**Pull request merged**